mlir/include/mlir/Dialect/Tosa/IR/TosaOps.td

```diff
@@ -1882,8 +1882,7 @@ def Tosa_PadOp : Tosa_InferShapedTypeOp<"pad"> {
   let arguments = (ins
     Tosa_RankedTensor:$input1,
     Tosa_Shape:$padding,
-    Optional<Tosa_ScalarTensor>:$pad_const,
-    OptionalAttr<I32Attr>:$input_zp
```

Comment:
Should this change be split into a separate commit, it seems orthognal to making the padding constant non optional.

Reply:
The upstreaming plan is to combine patches related to each operator into a single patch. Those two are the last for PadOp.